packages/assets-controllers/src/NftController.ts

```diff
@@ -188,16 +193,21 @@ export type NftControllerState = {
     };
   };
   ignoredNfts: Nft[];
+  isNftFetchingInProgress: {
+    [key: string]: { [chainId: Hex]: NftFetchStatus };
```

Comment:
I don't know whether it's important to reduce state size, but it could be reduced by `delete`ing these entries instead of setting them to false.

Reply:
I wasn't sure if its important or not either. It will be only one entry that we update per chain per account right, so i thought it would be okay to just override that value instead of deleting and then updating, wdyt?

Reply:
I agree a boolean is not my main concern about state size, so this is an optional optimization

Reply:
Resolving this because the logic was updated in this [commit](https://github.com/MetaMask/core/pull/4281/commits/85d2f52fc7764088392ba9d9e88f21e27b84f7d3) using the same approach used in tokenRatesController!